user-guide/Advanced_Functionality/DataMiner_Agents/Upgrading_a_DMA/Upgrade_Paths/Upgrade_path_960-96x_to_1030-103x.md

```diff
@@ -23,9 +23,16 @@ Make sure the IP network ports 9090, 4222, 6222, and 8222 (NATS monitoring only)
 > [!TIP]
 > See also: [Checking the required open ports in a DMS](xref:MOP_Checking_the_required_open_ports_in_a_DMS)
 
+## Running VerifyClusterPorts.dmupgrade
+
+Before you install the DataMiner upgrade package, [download VerifyClusterPorts.dmupgrade from DataMiner Dojo](https://community.dataminer.services/download/verifyclusterports-dmupgrade/) and run this package.
+
+> [!IMPORTANT]
+> If you do not run this package before executing the upgrade, the upgrade will fail. This is because DataMiner 10.3 assumes that NAS/NATS services are running, but they are not running in DataMiner 10.0.
+
```

Comment:
@FalconeDanielle, instead of adding the installation of VerifyClusterPorts.dmupgrade as an extra step under "Installing the DataMiner upgrade", I have copied the change you made in pull request #1372 and added it as a new subtitle. Is this OK?